packages/polymath-issuer/src/actions/compliance.js

```diff
@@ -239,7 +239,13 @@ export const exportWhitelist = () => async (
         ].join(',');
     });
 
-    window.location.assign(encodeURI(csvContent));
+    //Required to trigger file download in Chrome and Firefox
+    let link = document.createElement('a');
+    link.setAttribute('href', encodeURI(csvContent));
+    link.setAttribute('download', 'whitelist.csv');
+    document.body.appendChild(link);
+    link.click();
```

Comment:
This would be a massive security issue if this was possible! Imagine triggering a download from the code :p